src/main/java/de/tarent/maven/plugins/pkg/helper/Helper.java

```diff
@@ -1591,6 +1591,17 @@ public final String createDependencyLine(Set<Artifact> resolvedDependencies)
 		if (manualDeps.length() >= 2) {
 			manualDeps.delete(manualDeps.length() - 2, manualDeps.length());
 		}
+
+		/*
+		 * The user can override the defaultDependencyLine from the package-map
+		 * This is helpful for debian wheezy, when he wants openjdk-7-jre-headless
+		 */
+
+		if (targetConfiguration.getDefaultDependencyLine() != null &&
+		        !targetConfiguration.getDefaultDependencyLine().isEmpty()) {
+		    defaults = targetConfiguration.getDefaultDependencyLine();
+		}
```

Comment:
the whole conditional must definitely be moved up to line 1567 where the `defaults` is initially defined.